problem_builder/templates/html/dashboard.html

```diff
@@ -42,16 +42,30 @@
           {% for mcq in block.mcqs %}
             <tr>
               <th class="desc">{{ mcq.display_name }}</th>
-              <td class="value" {% if mcq.color %}style="border-right-color: {{mcq.color}};"{% endif %}>
-                {% if mcq.value %}{{ mcq.value }}{% endif %}
+              <td class="value"
+                  {% if mcq.color %} style="border-right-color: {{mcq.color}};"{% endif %}
+                  {% if not show_numbers %}
+                    {% if mcq.value %} aria-label="Score: {{mcq.value}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
+                  {% endif %}
+              >
+                {% if mcq.value and show_numbers %}
+                  {{ mcq.value }}
+                {% endif %}
               </td>
             </tr>
           {% endfor %}
           {% if block.has_average %}
            <tr class="avg-row">
-              <th class="desc">{% trans "Average" %}</th>
-              <td class="value" {% if block.average_color %}style="border-right-color: {{block.average_color}};"{% endif %}>
-                {{ block.average|floatformat }}
+              <th class="desc">{{ average_label }}</th>
+              <td class="value"
+                  {% if block.average_color %} style="border-right-color: {{block.average_color}};"{% endif %}
+                  {% if not show_numbers %}
+                    {% if block.average %} aria-label="Score: {{block.average|floatformat}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
```

Comment:
Next week is fine - "ASAP" as in, "as soon as prioritization possible" :)

On Thu, Apr 30, 2015 at 12:58 AM, Xavier Antoviaque <
notifications@github.com> wrote:

> In problem_builder/templates/html/dashboard.html
> <https://github.com/open-craft/problem-builder/pull/16#discussion_r29403780>
> :
>
> >                </td>
> >              </tr>
> >            {% endfor %}
> >            {% if block.has_average %}
> >             <tr class="avg-row">
> > -              <th class="desc">{% trans "Average" %}</th>
> > -              <td class="value" {% if block.average_color %}style="border-right-color: {{block.average_color}};"{% endif %}>
> > -                {{ block.average|floatformat }}
> > +              <th class="desc">{{ average_label }}</th>
> > +              <td class="value"
> > +                  {% if block.average_color %} style="border-right-color: {{block.average_color}};"{% endif %}
> > +                  {% if not show_numbers %}
> > +                    {% if block.average %} aria-label="Score: {{block.average|floatformat}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
>
> @sarina <https://github.com/sarina> Sure! Do you want it this week, or
> can it be done in next week's sprint? I would prefer the later if possible,
> as there will also likely be some additional follow-up fixes/improvements
> from Harvard, this way we could do everything at once. But let me know.
>
> Thanks for the review @cptvitamin <https://github.com/cptvitamin>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/open-craft/problem-builder/pull/16/files#r29403780>.
>